obal/data/roles/diff_package/tasks/main.yml

```diff
@@ -6,4 +6,4 @@
 - name: 'Set diff_package_changed to True when skipping package diffs'
   set_fact:
     diff_package_changed: True
-  when: diff_package_skip
+  when: diff_package_skip | bool
```

Comment:
Why that? This breaks `-e diff_package_skip=False`

Reply:
Ansible was yelling about this. Why does it break passing via `-e` ?

Reply:
`-e something=False` is a string, and the string "False" is true. (you'd have to pass `-e '{"something":false}'` for it to be a bool)